docs/hub/datasets-libraries.md

```diff
@@ -0,0 +1,16 @@
+# Libraries
+
+The Dataset Hub has support for several libraries in the Open Source ecosystem.
+Thanks to the `huggingface_hub` Python library, it's easy to enable sharing your datasets on the Hub.
+The Hub supports many libraries, and we're working on expanding this support!
+We're happy to welcome to the Hub a set of Open Source libraries that are pushing Machine Learning forward.
+
+The table below summarizes the supported libraries and their level of integration.
+
+| Library                                                                     | Description                                                                                                         | Download from Hub | Push to Hub |
+|-----------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|---|----|
+| [Dask](https://github.com/dask/dask) | Parallel and distributed computing library that scales the existing Python and PyData ecosystem.                                                           | ✅ | ✅ |
+| [Datasets](https://github.com/huggingface/datasets) | 🤗 Datasets is a library for accessing and sharing datasets for Audio, Computer Vision, and Natural Language Processing (NLP).              | ✅ | ✅ |
```

Comment:
Maybe make it more explicit that it's developed and supported by Hugging Face, unlike the other ones

Reply:
also: should we add "recommended" and move it to the first row?

Reply:
I think having the emoji is enough. Also no need to add `recommended` imo